sdcflows/workflows/syn.py

```diff
@@ -153,6 +151,8 @@ def init_syn_sdc_wf(omp_nthreads, epi_pe=None,
     # 1) BOLD -> T1; 2) MNI -> T1; 3) ATLAS -> MNI
     transform_list = pe.Node(niu.Merge(3), name='transform_list',
                              mem_gb=DEFAULT_MEMORY_MIN_GB)
+    transform_list.inputs.in3 = resource_filename(
+        'sdcflows', 'data/fmap_atlas_2_MNI152NLin2009cAsym_affine.mat')
```

Comment:
In reality, we only have this pre-calculated affine, so offering to switch was asking for trouble.